proposals/lip-0043.md

```diff
@@ -64,26 +64,22 @@ This property defines the set of eligible [BLS public keys][lip-0038] with their
+##### sidechainRegistrationFee
 
-The format of chain IDs aims to provide an efficient and compact way to uniquely identify chains in the ecosystem. This has another advantage in terms of usability: Users can easily remember the integer assigned as chain ID for their favorite blockchain applications.
+The `sidechainRegistrationFee` property accounts for the extra fee required to register the sidechain. It should be set to the value of the `REGISTRATION_FEE` constant.
 
-#### Sidechain Network Identifier
-
-The network identifier, or network ID, is a byte sequence unique to a chain that has to be prepended to the input of the signing function of every transaction, block, or message of the chain. It is necessary to avoid transaction replays between different chains in the ecosystem.
+#### Chain ID
 
-In the Lisk ecosystem, the network ID for a sidechain is computed as the hash digest of the sidechain genesis block ID given in a transaction with the registration command and the address of the account sending this transaction. This is convenient for two reasons:
+The chain ID is a 4-byte constant set in the chain configuration. Chain IDs serve two purposes: They are [prepended to the input of the signing function of every transaction, block, or message of the chain][lip-0037] to avoid transaction replays between different chains in the ecosystem; 
+They uniquely identify a chain in the Lisk ecosystem. In particular, in the Interoperability module, it serves a similar purpose for chains as [addresses][docs:address] do for user accounts, as it is used to identify the chain account in the Interoperability module store. Furthermore, the chain ID has to be stated in every cross-chain interaction. For example, it has to be specified in the `receivingChainID` property of a CCM to this sidechain and in the `sendingChainId` property of a cross-chain update command from this sidechain.
 
-1. The sidechain developers can pre-compute the network ID of their sidechain as soon as they set the genesis block and an account to send the transaction on the mainchain. The sidechain can be started already from this moment without being registered on the mainchain.
-2. The network ID is known to the mainchain as soon as the sidechain is registered, thus it can validate cross-chain update commands coming from the sidechain without further context.
+ The sidechain chain ID is given as a parameter in the sidechain registration command: If the given value is already taken by another sidechain, the sidechain registration command fails. In this case, the sidechain has to change the chain ID with an hardfork, and resubmit the sidechain registration command with a new value.
```

Comment:
yes, the same is true for the name. However, this is less of an issue as it does not require an hardfork in the sidechain (since the name, normally, it s not used for anything in the protocol)

Reply:
Ok, so I would then just mention this here explicitly

Reply:
@gkoumout If we should make an edit here, could you make a suggestion for it?

Reply:
Can be resolved.